index.d.ts

```diff
@@ -128,6 +133,14 @@ export const Either: TypeRep;
 export function Left<A, B>(value: A): Either<A, B>;
 export function Right<A, B>(value: B): Either<A, B>;
 
+//  Placeholder
+
+interface Placeholder {
+  '@@functional/placeholder': boolean
```

Comment:
I think you could simply use `true` instead of `boolean`. Wouldn't want Sanctuary to treat my value as a placeholder even though I *explicitly* stated that `@@function/placeholder` is `false`!